README.md

```diff
@@ -1,192 +1,61 @@
-
-## Usage Example
-
-### Initialize & Register
-
-```javascript
-import { EThree } from '@virgilsecurity/e3kit'
-// get virgil token from you backend (better to protect it!)
-const getToken = () => fetch('http://localhost:3000/get-virgil-jwt/')
-    .then(res => res.json())
-    .then(data =>  data.token);
-
-(async function() {
-    // get your unique identity from backend
-    const sdk = await EThree.initialize(getToken);
-    // create private key and upload it to our protected cloud service
-    await sdk.register();
-    await sdk.backupPrivateKey('encryption_pwd');
-})();
-```
-
-### Encrypt & Decrypt
-
-```javascript
-const usersToEncryptTo = ['alice@myapp.com', 'bob@myapp.com', 'sofia@myapp.com'];
-const userWhoEncrypts = 'alex@myapp.com';
-const [receiverPublicKeys, senderPublicKey] = await Promise.all([
-    eThree.lookupPublicKeys(usersToEncryptTo),
-    eThree.lookupPublicKeys(userWhoEncrypts)
-]);
-
-const encryptedMsg = await eThree.encrypt('Send you my sensitive information!', receiversPublicKeys);
-const decryptedMsg = await eThree.decrypt(encryptedMsg, senderPublicKey);
-// we decrypt the message and check that it is sent by 'alex@myapp.com'
-```
-
-### Enable Group Chat
-
-In this section, you'll find out how to build a group chat using the Virgil E3Kit.
-
-We assume that your users have installed and initialized the E3Kit, and used snippet above to register.
-
-
-#### Create group chat
-
-Let's imagine Alice wants to start a group chat with Bob and Carol. First, Alice creates a new group ticket by calling `createGroup` method and E3Kit stores the ticket in Virgil Cloud. This ticket holds a shared root key for future group encryption.
+<a href="https://developer.virgilsecurity.com/docs"><img width="100px" src="https://cdn.virgilsecurity.com/assets/images/github/logos/e3kit/E3Kit.png" align="left" hspace="10" vspace="6"></a> [Virgil Security](https://virgilsecurity.com) provides the [E3Kit](https://developer.virgilsecurity.com/docs/e3kit/) - an open-source client-side framework that allows developers to add end-to-end encryption to their messaging applications, file sharing programs, and other digital communication products in just a few simple steps to become HIPAA and GDPR compliant and more.
 
-Alice has to specify a unique `identifier` of group with length > 10 and `findUsersResult` of participants. We recommend tying this identifier to your unique transport channel id.
+## Benefits
 
-```javascript
-const groupId = 'unique_group_id';
-const participants = await eThree.findUsers(['bob@myapp.com', 'carol@myapp.com']);
-const group = await eThree.createGroup(groupId, participants);
-// Group created and saved locally
-```
+- Easy to setup and integrate into new or existing projects
+- Compatible with any CPaaS provider, including Nexmo, Firebase, Twilio, PubNub and etc.
+- Strong secret keys storage, integration with Keychain
```

Comment:
Why Keychain is highlighted? It isn't main JS storage and we support all types of storages: localstorage for browsers, filesystem for node, keychain for ios react native and keystore for android

Reply:
@MariiaMalitska